src/Makefile

```diff
@@ -1950,6 +2000,14 @@ ifneq ($(wildcard external/*/*),)
 ifneq ($(wildcard external/libdb/build_unix/*),)
 	cd ${EXTERNAL_LIBDB} && ${MAKE} realclean
 endif
+
+ifneq ($(wildcard external/libarchive-3.5.1/*),)
+	cd ${EXTERNAL_LIBARCHIVE} && ${MAKE} clean
+endif
+
+ifneq ($(wildcard external/pacman-5.2.2/*),)
+	cd ${EXTERNAL_PACMAN} && ${MAKE} clean
```

Comment:
Hi! This part of the `clean` rule is reporting an error, as there's no Makefile with `clean` rule inside the `pacman-5.2.2 `directory:

```
cd external/pacman-5.2.2/ && make clean
make[1]: Entering directory '/home/vagrant/wazuh/src/external/pacman-5.2.2'
make[1]: *** No rule to make target 'clean'.  Stop.
make[1]: Leaving directory '/home/vagrant/wazuh/src/external/pacman-5.2.2'
make: *** [Makefile:2001: clean-external] Error 2
```